_posts/2017/2017-02-20-continuous-documentation-with-ansible.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: post
+title: "Continuous Documentation with Ansible"
```

Comment:
From our analytics tools we've learnt, that at a certain point people don't read any further and do prefer the shorter posts. So I highly recommend to split it up. We've done that with the Elixir posts and will do with the Sales App. Which worked out pretty fine.